hc/api/tests/.cache/v/cache/lastfailed

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
I am not quite sure how this file came by, but I think it ought to be kept away from the repository.


**Resource**: https://goo.gl/RBRn6p